internal/backends/sqlite/metadata/registry.go

```diff
@@ -179,7 +235,10 @@ func (r *Registry) CollectionDrop(ctx context.Context, dbName string, collection
 		return false, nil
 	}
 
-	tableName := r.CollectionToTable(collectionName)
+	tableName, err := r.CollectionGet(ctx, dbName, collectionName)
+	if err != nil {
+		return false, nil
```

Comment:
It's not enough to just return nil in this case, e.g. if the query failed because `rows.Scan` returned an error, we should have a lazy error here.

Reply:
Nice catch!